spec-0323-vestigial-clamps.md

```
@@ -0,0 +1,42 @@
+# Spec: Remove Vestigial .min(u32::MAX) Clamps — work-803dfbba
+
+## Feature/Behavior Description
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)